microservices/embeddingservice/src/main.py

```diff
@@ -13,7 +13,8 @@
 CONFIG_PATH = os.environ.get("CONFIG_FILE", default="config.yaml")
 
 config = EnvYAML(CONFIG_PATH)
-text_embedder = EmbedText(config)
+model_config = config.get("models", default="")
```

Comment:
I think a default is the right way to go here, as no models renders the service useless